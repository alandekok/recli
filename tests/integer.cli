int 1
int -1
int -2.3
int 0x44
int 0334
int "hello"
int 484747
