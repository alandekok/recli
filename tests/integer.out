int -2.3
    ^ Parse error.
int 0x44
    ^ Parse error.
int "hello"
    ^ Parse error.
